tests/general/container_test.go

```diff
@@ -90,7 +90,7 @@ func TestSpecifiedContainerConfigDefaultsToCmdLineArgIfEnvVarConflict(t *testing
 			if strings.Contains(
 				log.Message,
 				`Both environment variable SPLUNK_CONFIG and flag '--config' were specified. `+
-					`Using the flag value /etc/config.yaml and ignoring the environment variable value `+
+					`Using the flag values and ignoring the environment variable value `+
```

Comment:
Is logging the values no longer desirable?

Reply:
There's another log message that shows up directly after this that includes the values that `config` is set to. Here's the shell output:

```
crobert@ ~/dev/splunk-otel-collector $ ./bin/otelcol_darwin_amd64 --config processlistconfig.yaml --config services.yml 
2022/05/23 15:26:44 main.go:184: Both environment variable SPLUNK_CONFIG and flag '--config' were specified. Using the flag values and ignoring the environment variable value /etc/fake_config.yaml in this session
2022/05/23 15:26:44 main.go:222: Set config to [processlistconfig.yaml,services.yml]
```
It seemed unnecessary to have the flag values for both log messages, and more consistent if we just log what the config sources are every time, regardless if it's coming from an environment variable or command line flag.

Let me know if you think it's still valuable to have both though!